akka-docs/src/main/paradox/remoting.md

```diff
@@ -2,10 +2,13 @@
 
 For an introduction of remoting capabilities of Akka please see @ref:[Location Transparency](general/remoting.md).
 
+We recommend that you prefer @ref:[Akka Cluster](cluster-usage.md) over using remoting directly. As remoting is the
+underlying module that allows for Cluster, it is still useful to understand details about it though.
+
 @@@ note
 
 As explained in that chapter Akka remoting is designed for communication in a
-peer-to-peer fashion and it has limitations for client-server setups. In
+peer-to-peer fashion and it is not a good fit for client-server setups. In
```

Comment:
👍

Reply:
same change in remoting-artery docs?

Reply:
Artery has a bit different wording and is already pretty clear about this.